samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c

```diff
@@ -56,7 +56,7 @@ BT_MESH_MODEL_PUB_DEFINE(light_lightness_cli_pub, NULL, 2 + 5);
 BT_MESH_MODEL_PUB_DEFINE(light_ctl_srv_pub, NULL, 2 + 9);
 BT_MESH_MODEL_PUB_DEFINE(light_ctl_cli_pub, NULL, 2 + 9);
 
-BT_MESH_MODEL_PUB_DEFINE(vnd_pub, NULL, 3 + 2);
+BT_MESH_MODEL_PUB_DEFINE(vnd_pub, NULL, 3 + 6);
```

Comment:
Unrelated to this PR, but why do you have a vendor model in this app? I think it may confuse people who are only looking for a reference for standard models.

Reply:
It is related. I'm going to rename main heading.

Developers are not going to depends on only Standard Models. My goal is to explore all
capabilities under one demo.